internal/proto/networkfencing.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+package proto;
+
+option go_package = "github.com/csi-addons/kubernetes-csi-addons/internal/proto";
+
+// NetworkFencing holds the RPC method for allowing the communication between
+// the CSIAddons controller and the sidecar for fencing operations.
+service NetworkFencing {
+    // ControllerReclaimSpace is a procedure that gets called on the CSI
+    // sidecar.
+    rpc FenceClusterNetwork (FenceRequest) returns(FenceResponse) {}
+
```

Comment:
It will be good to have comments which explains a bit about FenceClusterNetwork and UnFenceClusterNetwork.